ci/infra/azure/network.tf

```diff
@@ -66,3 +66,22 @@ resource "azurerm_private_dns_zone_virtual_network_link" "internal_dns" {
   virtual_network_id    = azurerm_virtual_network.virtual_network.id
   registration_enabled  = true
 }
+
+# Route Table
+resource "azurerm_route_table" "nodes" {
```

Comment:
Are route resources here for managing routing rules by azure cloud provider?

Reply:
Yes need to exist for the CCM.